src/praisonai-code/praisonai_code/cli/commands/code.py

```
@@ -316,6 +317,49 @@ def code_main(
+    session's args (model/workspace/no-acp/no-lsp) onto ``AsyncTUIConfig``.
+    """
+    import os
+
+    from praisonai_code.cli.interactive.async_tui import AsyncTUI, AsyncTUIConfig
+
+    model = getattr(args, "llm", None)
+    try:
+        from ..configuration.model_resolver import resolve_default_model
+
+        resolved_model = resolve_default_model(model)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)